corehq/form_processor/tests/utils.py

```diff
@@ -170,6 +172,10 @@ def _delete_all(db, viewname, **view_kwargs):
 )
 
 
+def sql_backend_case(cls):
```

Comment:
@benrudolph you can use [`nose.tools.nottest`](http://nose.readthedocs.io/en/latest/testing_tools.html#nose.tools.nottest) to decorate things that are not tests but nose thinks they are.